tests/e2e/pact/broker/verify.ts

```diff
@@ -123,7 +129,54 @@ async function verifyTracker(): Promise<void> {
   await verifyOnce("tracker")
 }
 
+function clearData() {
+  if (isSandbox) return
+  const releaseUrl = `${process.env.PACT_PROVIDER_URL}${basePath}/Task/$release`
+
+  const nominatedReleases = fetcher.taskExamples
+    .filter(task => task.isSuccess)
+    .filter(task => task.requestFile.operation === "release")
+    .map(task => task.request as fhir.Parameters)
+    .filter(isNominatedRelease)
+
+  nominatedReleases.forEach(async release => {
+    let response
+    do {
+      console.log(
+        "Clearing Prescriptions For: ",
+        getIdentifierParameterByName(release.parameter, "owner").valueIdentifier.value
+      )
+      response = await nominatedRelease(releaseUrl, release)
+    }
+    while (response && response.data.resourceType !== "OperationOutcome")
```

Comment:
Might be nice to make this condition more specific. We could look for the "no more prescriptions available" issue code.

Reply:
Listening to me is generally a bad idea